cypress/e2e/Mock User to Test and Project.cy.ts

```diff
@@ -0,0 +1,37 @@
+describe('Mocking User and Test Project', () => {
+  it('Mock User and Test Project', () => {
+
+    // Stub login request
+    cy.intercept('GET', 'http://localhost:4173/api/v1/user', {
+      statusCode: 200,
+      body: {
+        id: '756dae7c-6677-4ba3-8fb5-863bcea1b18c',
+        name: 'Cypress Hill',
+        email: 'Cypress@Hill.de',
+        registerDate: '2024-06-11',
+        lasLoginDate: '2024-06-11T15:54:23'
```

Comment:
There seems to be a typo in lastLoginDate

Reply:
True, i fixed this in the latest commit. It is now listet as _lastLoginDate: '2024-06-11T15:54:23'_